qiling/core.py

```diff
@@ -206,6 +206,14 @@ def __init__(
         # Add extra guard options when configured to do so
         self._init_stop_guard()
 
+        ###############################
+        # Properties configured later #
+        ###############################
```

Comment:
all these could be in os ?

Reply:
This is merely documenting which properties are set by various other functions later on :)

Reply:
i am trying to "slim" down core.py. and move os/arch related properties out from core.py

Reply:
ah, i might misunderstand what you trying to do. 

again, what is this for.

Reply:
Ok, i guess we move this up after definition, line 85